solidity/contracts/bridge/Bridge.sol

```diff
@@ -109,6 +109,15 @@ contract Bridge is Ownable {
         uint32 sweptAt;
     }
 
+    // TODO: Documentation.
+    struct RedemptionRequest {
+        address redeemer;
+        uint64 requestedAmount;
+        uint64 redeemableAmount;
```

Comment:
What is `redeemableAmount`? The max amount redeemable is `requestedAmount`, so I thought we just need to know the minimum?

Reply:
It's not that simple. Suppose a redeemer requests 5 TBTC (`requestedAmount`). The redemption fee paid to the treasury is 0.2 TBTC. The maximum transaction fee is 0.1 BTC. The Bridge must take 5 TBTC from the redeemer balance upon request. Upon proof submission, the bridge must put away the redemption fee so only 4.9 TBTC (`redeemableAmount`) can be burned from the bridge balance. In fact, the maximum value which can be redeemed assuming there are no tx fees is 4.9 TBTC. Taking the max fee into account gives 4.7 BTC as `minimalAmount`. However, in case of timeout, you must return the entire 5 TBTC to the redeemer. Hence we need to store those three values.

Reply:
Done in https://github.com/keep-network/tbtc-v2/pull/128/commits/2da1001354b25771fc94494febb511a5bccaec5d